skgrid/tools/skgrid-mlib.cwl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
```

Comment:
Is it 1.0 or 1.2? I don't remember which features got added for 1.2, but might want to just bump the version to be safe (I'm assuming you are testing against a current version of CWLTool, which would be deploying 1.2)

Reply:
deploying against v1.0

looks like v1.2 removed fields so pipeline is not longer valid in 1.2 and workflow breaks if updating version. Can update to v1.2 in future PR